### PR TITLE
CO : Add some channels

### PR DESCRIPTION
**Pull request by BellezaEmporium**
Here's a few channel additions.

Website link and LyngSat Stream link available on each commit.

### CODE REVIEW

File: channels/co.m3u [hunk 1]
```diff
@@ -45,7 +49,13 @@ https://telecaribe-deportes.gcdn.anvato.net/hls/live/telecaribedeporteshd2/3296k
...
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://www.lyngsat-stream.com/logo/tv/tt/telepacifico_co.png" group-title="",Telepacifico
+https://stream.logicideas.media/telepacifico-live/smil:live.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://i.imgur.com/eg47wmv.jpg" group-title="",Televid (486p)
 https://api.new.livestream.com/accounts/28238881/events/8574038/live.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://live-tv-channels.org/pt-data/uploads/logo/co-tv-gracia.jpg" group-title="",TV Gracia
+https://streamyes.alsolnet.com/tvgracia/live/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://i.imgur.com/kady79t.jpg" group-title="",UAO Play
 https://live-edge-rbx-1.cdn.enetres.net/184784E1D210401F8041E3E1266822CC021/live-1500/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://www.zoomcanal.com.co/templates/rt_callisto/custom/images/logos/zoom/ZOOM%20SOLO%20+%20PNG.png" group-title="",Zoom TV
+https://player.cdnmedia.tv/embed/undefined/canalzoomtvlive/smil:especialesweb01.smil/playlist.m3u8
```
Comment (freearhey) [inline channels/co.m3u:61]: The link no longer works.
Captura de pantalla 2021-01-24 a las 4 52 02

**Review by freearhey — CHANGES_REQUESTED** [review]

File: channels/co.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/co.m3u:61]: I'm going to have a look at it. Might not be 24/7.

Comment (LaneSh4d0w) [reply channels/co.m3u:61]: Another link : https://player.cdnmedia.tv/embed/undefined/canalzoomtvlive/smil:especialesweb02.smil/playlist.m3u8

Comment (LaneSh4d0w) [reply channels/co.m3u:61]: What's interesting is, they got 3 links for their channel.

image

Comment (LaneSh4d0w) [reply channels/co.m3u:61]: I'm deleting it, it's giving me a blank page when trying.

**Review by freearhey — APPROVED** [review]

**Pull request merged**